### DISCUSSION TITLE
Support for Flutter

### DISCUSSION THREAD

**Discussion by sheikh-iffi** [tangrams/tangram-es#2289] [Q&A]
Hi everyone,

Is there a way to use Tangram ES for flutter ?

**Comment by matteblair** [comment]
Hello! As far as I know, there is no existing integration for the Flutter SDK. 

I am not very familiar with Flutter, but in theory it should be possible to build a Flutter plugin for Tangram ES. I have no plans to work on this, but I would be happy to assist if someone else wanted to contribute this work!
